### PR TITLE
Add YFARM space

### PR DESCRIPTION
**Pull request by Davidyffs**
**Add space for YFARM**  
  
Etherscan: https://etherscan.io/token/0xF5D0FefAaB749d8B14C27F0De60cC6e9e7f848d1
CoinGecko: https://www.coingecko.com/en/coins/yfarm-token
Website: https://farm.yffs.finance/
Twitter: https://twitter.com/yffsfinance

### CODE REVIEW

File: spaces/yfarm/index.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "key": "yfarm",
+  "symbol": "YFARM",
+  "name": "FARM.YFFS.FINANCE",
```
Comment (e-asia) [inline spaces/yfarm/index.json:4]: Can you choose a shorter name? This will break the interface otherwise.

Comment (e-asia) [reply spaces/yfarm/index.json:4]: Or add space between words.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/yfarm/index.json (hunk 1 above)
Comment (e-asia) [reply spaces/yfarm/index.json:4]: image

Comment (Davidyffs) [reply spaces/yfarm/index.json:4]: short 
name is YFARM

Comment (Davidyffs) [reply spaces/yfarm/index.json:4]: just YFARM please

Comment (e-asia) [inline spaces/yfarm/index.json:4]: Not done yet.
```suggestion
  "name": "FARM YFFS FINANCE",

```

```suggestion
  "name": "YFFS.FINANCE",

```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Review by e-asia — APPROVED** [review]

**Pull request merged**